### ISSUE TITLE
handle 409 error in settings flow

### ISSUE THREAD

**Issue by williamohara** [ory/kratos#1107]
**Is your feature request related to a problem? Please describe.**

if a user tries to sign up with the same identifier twice using the registration flow, a message is returned 
`
id: 4000007
text: "An account with the same identifier (email, phone, username, ...) exists already."
type: "error"
`
if a user tires to update their identifier in the settings flow, the error flow is started returning a less descript error message

`
code: 409
message: "Unable to insert or update resource because a resource with that value exists already"
status: "Conflict"
`
**Describe the solution you'd like**

handle the 409 error in the settings flow and make it parallel to the registration flow. Return a message in the settings flow under the methods.profile.config.messages 

**Describe alternatives you've considered**

**Additional context**

I am using the Admin endpoint for settings flow i tried with the public endpoint and same

**Comment by aeneasr** [comment]
Yup, that should be fixed!

**Comment by williamohara** [comment]
I am building an authentication service for my project on top of kratos. I am going to integrate with hydra and possibly keto too. I have built my auth FE app from scratch and I think that it would be a good idea to get to know your code a bit better AND learn Go ( I am just beginning to learn it) I think that this would be a good one as you already have messages for other flows - it would be just repeating the pattern. I'm going to fork and give it a try. Hopefully you will see a pull request soon. I have never done an open source contribution before either 😮 so it will be my first.

**Comment by aeneasr** [comment]
Awesome! I think it will be a great learning experience! And you can't do anything wrong - just give it your best try and we'll continuously improve it :)

**Comment by williamohara** [comment]
Seems to only be a problem when using MySQL

**Comment by williamohara** [comment]
@aeneasr I finally got back to this. I set up my dev environment and built off of the latest code - and I could not reproducer this bug. I looked back at the commits since the last release.. (docker pull oryd/kratos:latest-sqlite) and it seems that this issue is a duplicate of this issue https://github.com/ory/kratos/issues/694. Which was already fixed..

I guess I'll need to find another way to contribute :)
